### DISCUSSION TITLE
How to setup exo on kubernetes

### DISCUSSION THREAD

**Discussion by hclx-sohail** [exoplatform/exo-community#69] [Q&A]
Hello everyone,
We had a requirement to setup exo platform on our kubernetes cluster. Everywhere over the internet we could only find methods to setup exo on VMs. So we collected all the dockerfiles, created manifests and deployed on our clusters. But the platform is not stable as expected. Sometimes we face login issues, sometimes unable to create new users and sometimes data is lost. Everytime we have to restart the pods. It is so sudden that we are unable to find the exact reason for this behaviour.

I request from the community if anyone has any experience, documents, articles or walkthroughs to properly setup exo platform on kubernetes.

Thanks

**Comment by rdenarie** [comment]
Hello, 
We recently try to run eXo on K8s, but we faced the same instability as the ones you list.

We haven't a functionnal configuration currently to run eXo in K8s, and we are working on that.
I'll notify you when we reach a satisfying level.

**Comment by alexoberco** [comment]
Hi, there is any update on this? I have my own cluster and it would be interesting to test it here!!
